vault/data_source_ldap_static_role_credentials.go

```diff
@@ -119,6 +124,15 @@ func readLDAPStaticCreds(ctx context.Context, d *schema.ResourceData, meta inter
 	if err := d.Set(consts.FieldUsername, response.username); err != nil {
 		return diag.FromErr(err)
 	}
+	if provider.IsAPISupported(meta, provider.VaultVersion116) {
+		var skip bool
+		if skipRaw, ok := secret.Data[consts.FieldSkipImportRotation]; ok {
```

Comment:
Does the `GET /ldap/static-cred/:role_name` endpoint return this field in the response? I don't see it documented: https://developer.hashicorp.com/vault/api-docs/v1.16.x/secret/ldap#static-role-passwords

Reply:
yeah, i think i misunderstood when i saw some fields in the role are included in the cred return.